.github/workflows/onPush.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [push]
+
+jobs:
+  unitTests:
+    name: unitTests
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.13.0"
```

Comment:
Is there a way this version can be dynamic? As in anything greater than or equal to 12.13.0?

Reply:
We could use a different build step for each target version of node, yes.  I'm not sure how far into that rabbit hole we want to go.